src/Client.js

```diff
@@ -1,8 +1,10 @@
 const amqp = require('./network/amqp');
+const http = require('./network/http');
```

Comment:
For reasons already mentioned, maybe this should be `HTTP` and use es6 imports :point_down: 

```suggestion
import HTTP from './network/http';
```

Reply:
Done!